src/monitor.rs

```diff
@@ -68,11 +68,9 @@ impl VideoMode {
         self.video_mode.bit_depth()
     }
 
-    /// Returns the refresh rate of this video mode. **Note**: the returned
-    /// refresh rate is an integer approximation, and you shouldn't rely on this
-    /// value to be exact.
+    /// Returns the refresh rate of this video mode in mHz.
     #[inline]
-    pub fn refresh_rate(&self) -> u16 {
+    pub fn refresh_rate(&self) -> u32 {
```

Comment:
Is this a "silent break change" (changing the units without changing the API, other than the return type)? It might then be preferable to change the name, e.g. to `refresh_rate_mHz`?

Besides which, what is the purpose of exporting this value? Is it used to schedule frames (probably not the best mechanism?), or only to get a rough idea of monitor performance? In the latter case, this is unnecessary accuracy.

Reply:
The idea is to have a fallback to schedule frames when vsync extension isn't present and we don't have any better way to do so. Aslo most APIs return it as mHz on Linux which makes sense, since your refresh rate is something like 59.997 most of the time.

Reply:
> Is this a "silent break change"

I think that's likely, yeah. Creating a new method and deprecating the old one is probably the way to go.

Reply:
I think we can safely remove the old one given that there's no need for it anymore? They are the same, just precision got changed.